Packs/DeHashed/Integrations/DeHashed/DeHashed.py

```diff
@@ -225,6 +236,47 @@ def dehashed_search_command(client: Client, args: Dict[str, str]) -> tuple:
         )
 
 
+def email_command(client: Client, args: Dict[str, str]) -> tuple:
+    """
+    This command returns data regarding a compromised email address
+    :param client: Demisto client
+    :param args:
+    - email: the email address that should be checked
+    :return: Demisto outputs
+    """
+    email_address = argToList(args.get('email'))
+    result = client.dehashed_search('email', email_address, 'contains')
+    if not isinstance(result, dict):
+        raise DemistoException(f"Got unexpected output from api: {result}")
+
+    query_data = result.get("entries")
+    if not query_data:
+        context = {
+            'DBotScore':
+                {
+                    'Indicator': email_address[0],
+                    'Type': 'email',
+                    'Vendor': INTEGRATION_CONTEXT_BRAND,
+                    'Score': 0
+                }
+        }
+        return "No matching results found", context, None
+    else:
+        default_dbot_score_email = 2 if client.email_dbot_score == 'SUSPICIOUS' else 3
```

Comment:
you need to check here if the exact mail is found, because we are using `contained`.
for example:
if you search for alice@gmail.com, you may find that prefix_alice@gmail.com will be found. but it is not the same mail.

please add a check here that the exact mail is in the results and please add a unit test for that.
return 0 if not.